### PR TITLE
+openjdk.org@21

### PR DESCRIPTION
**Pull request by felipecrs**

### CODE REVIEW

File: projects/openjdk.org/package.yml [hunk 1]
```diff
@@ -144,6 +152,8 @@ provides:
   - bin/jstack
   - bin/jstat
   - bin/jstatd
+  # TODO: if: {{version.major}} >= 21
+  # - bin/jwebserver
```
Comment (felipecrs) [inline projects/openjdk.org/package.yml:156]: Any ideas about this?

Comment (jhheider) [reply projects/openjdk.org/package.yml:156]: just need to omit it; it would be a potential upgrade to libpkgx if desired.

**Comment by jhheider** [comment]
great addition. thanks!

File: projects/openjdk.org/package.yml (hunk 1 above)
Comment (felipecrs) [reply projects/openjdk.org/package.yml:156]: https://github.com/pkgxdev/libpkgx/issues/68

**Pull request merged**